scripts/demo.py

```diff
@@ -41,7 +42,7 @@ def demo(config):
     image = Image.open(config.input_pic).convert('RGB')
     images = transform(image).unsqueeze(0).to(device)
 
-    model = get_model(args.model, pretrained=True, root=args.save_folder).to(device)
+    model = get_model(args.model,local_rank=args.local_rank,pretrained=True, root=args.save_folder).to(device)
```

Comment:
`model = get_model(args.model, local_rank=args.local_rank, pretrained=True, root=args.save_folder).to(device)`

Reply:
okay, I got it!